File: pi.py
#!/usr/bin/env python3
"""
Created by: Mr. Logan
Created on: February 2025
This is the "Radius to Circumference" module
"""

import math


def main() -> None:
    """The main() function gets circumference from radius, returns None."""


radius = float(input("Insert radius(cm): "))
pi = math.pi
print(f"If the radius of a circle is {radius} circumference of the radius is {round(radius*pi*2, 5)} cm")


if __name__ == "__main__":
    main()

File: pi.c
// Copyright (c) 2020 Mr. Coxall All rights reserved
//
// Created by: Mandela Logan
// Created on: February 2025
// This program calculates the area of a circle
//    with radius 15mm

#include <stdio.h>
#include <math.h>

int main()
{
    int radius, circumference; // Declaring variables to store two numbers and their product

    // Asking the user to input two numbers
    printf("Enter radius: ");
    scanf("%d", &radius);

    // Calculating the product of two numbers
    circumference = radius * M_PI * 2;

    // Printing the result
    printf("Circumference: %d", circumference);
    printf("\n");

    return 0; // Indicating that the program ended successfully
}